include/constants/interactables.h

```diff
@@ -72,7 +72,7 @@
 #define IA__ICE_PARADISE__HALFPIPE__START          68
 #define IA__069                                    69
 #define IA__070                                    70
-#define IA__071                                    71 // Interactable 071 calls the code of IA__044
+#define IA__071                                    71 // MapEntity 071 calls the code of IA__044
```

Comment:
I guess these comments don't need to be replaced?

Reply:
Ah damn... I used case-sensitive auto-replace with full names...